genadrop/test_home_page.py

```diff
@@ -82,14 +87,39 @@ def test_header_mint_button(self):
 
         # Verify tab title
         actual_tab_title = self.driver.title
-        assert actual_tab_title == self.MINT_TAB_TITLE,\
+        assert actual_tab_title == self.MINT_TAB_TITLE, \
             f"Unexpected tab title for Mint page, actual: '{actual_tab_title}', expected: '{self.MINT_TAB_TITLE}'"
 
         # Verify page main area title
         actual_page_title = self.wait.until(EC.visibility_of_element_located((By.TAG_NAME, 'h1'))).text
-        assert actual_page_title == self.MINT_PAGE_TITLE,\
+        assert actual_page_title == self.MINT_PAGE_TITLE, \
             f"Unexpected page title on Mint page, actual: '{actual_page_title}', expected: '{self.MINT_PAGE_TITLE}'"
 
+    def test_near_foundation_link(self):
+        """ TC id: GD_HP008 """
+        near_foundation_link = self.wait.until(
+            EC.visibility_of_element_located((By.XPATH, "//img[@class='Orgs_org__2zmxJ'][2]")))
+        near_foundation_link.click()
+        window_name = self.driver.window_handles[-1]
```

Comment:
would be helpful to add verification that new tab opened and there're 2 tabs (window_handles returns a list with 2 elements)